diff -up cups-filters-1.28.11/utils/driverless.c.omv~ cups-filters-1.28.11/utils/driverless.c
--- cups-filters-1.28.11/utils/driverless.c.omv~	2022-01-16 16:54:20.139702481 +0100
+++ cups-filters-1.28.11/utils/driverless.c	2022-01-16 16:56:40.882250177 +0100
@@ -284,17 +284,17 @@ listPrintersInArray(int reg_type_no, int
 	   strcasestr(pdl, "image/"))) {
 	value[0] = '\0';
 	if (strcasestr(pdl, "application/pdf"))
-	  strncat(value, ",PDF", sizeof(value));
+	  strncat(value, ",PDF", sizeof(value)-strlen(value)-1);
 	if (strcasestr(pdl, "application/PCLm"))
-	  strncat(value, ",PCLM", sizeof(value));
+	  strncat(value, ",PCLM", sizeof(value)-strlen(value)-1);
 	if (strcasestr(pdl, "application/postscript"))
-	  strncat(value, ",PS", sizeof(value));
+	  strncat(value, ",PS", sizeof(value)-strlen(value)-1);
 	if (strcasestr(pdl, "application/vnd.hp-PCL"))
-	  strncat(value, ",PCL", sizeof(value));
+	  strncat(value, ",PCL", sizeof(value)-strlen(value)-1);
 	if (strcasestr(pdl, "image/pwg-raster"))
-	  strncat(value, ",PWGRaster", sizeof(value));
+	  strncat(value, ",PWGRaster", sizeof(value)-strlen(value)-1);
 	if (strcasestr(pdl, "image/urf"))
-	  strncat(value, ",AppleRaster", sizeof(value));
+	  strncat(value, ",AppleRaster", sizeof(value)-strlen(value)-1);
 	for (ptr = strcasestr(pdl, "image/"); ptr;
 	     ptr = strcasestr(ptr, "image/")) {
 	  char *valptr = value + strlen(value);
