diff -up cups-filters-2.0.0/utils/driverless.c.1~ cups-filters-2.0.0/utils/driverless.c
--- cups-filters-2.0.0/utils/driverless.c.1~	2023-09-23 22:28:14.327198057 +0200
+++ cups-filters-2.0.0/utils/driverless.c	2023-09-23 22:30:02.617897457 +0200
@@ -275,17 +275,17 @@ listPrintersInArray(int reg_type_no, int
 	   strcasestr(pdl, "image/"))) {
 	value[0] = '\0';
 	if (strcasestr(pdl, "application/pdf"))
-	  strncat(value, ",PDF", sizeof(value) - 1);
+	  strncat(value, ",PDF", sizeof(value) - strlen(value) - 1);
 	if (strcasestr(pdl, "application/PCLm"))
-	  strncat(value, ",PCLM", sizeof(value) - 1);
+	  strncat(value, ",PCLM", sizeof(value) - strlen(value) - 1);
 	if (strcasestr(pdl, "application/postscript"))
-	  strncat(value, ",PS", sizeof(value) - 1);
+	  strncat(value, ",PS", sizeof(value) - strlen(value) - 1);
 	if (strcasestr(pdl, "application/vnd.hp-PCL"))
-	  strncat(value, ",PCL", sizeof(value) - 1);
+	  strncat(value, ",PCL", sizeof(value) - strlen(value) - 1);
 	if (strcasestr(pdl, "image/pwg-raster"))
-	  strncat(value, ",PWGRaster", sizeof(value) - 1);
+	  strncat(value, ",PWGRaster", sizeof(value) - strlen(value) - 1);
 	if (strcasestr(pdl, "image/urf"))
-	  strncat(value, ",AppleRaster", sizeof(value) - 1);
+	  strncat(value, ",AppleRaster", sizeof(value) - strlen(value) - 1);
 	for (ptr = strcasestr(pdl, "image/"); ptr;
 	     ptr = strcasestr(ptr, "image/")) {
 	  char *valptr = value + strlen(value);
